frontend/src/index.jsx

```diff
@@ -12,16 +12,16 @@ import theme from './theme'
 
 const root = ReactDOM.createRoot(document.getElementById('root'))
 root.render(
-  <React.StrictMode>
-    <ChakraProvider theme={theme}>
-      <ColorModeScript initialColorMode={theme.config.initialColorMode} />
-      <HelmetProvider>
-        <AuthProvider>
-          <App />
-        </AuthProvider>
-      </HelmetProvider>
-    </ChakraProvider>
-  </React.StrictMode>
+  // <React.StrictMode>
```

Comment:
Curious why strict mode is disabled and if the root cause can be resolved

Reply:
Was doing some testing w/o it, I will re-enable this, it should not affect anything